components/UI/backButton.tsx

```diff
@@ -7,11 +7,29 @@ type BackButtonProps = {
   onClick: () => void;
 };
 
-const BackButton: FunctionComponent<BackButtonProps> = ({ onClick }) => (
-  <button onClick={onClick} className={styles.backButton}>
-    <ArrowLeftIcon color={theme.palette.secondary.main} width="24" />
-    <p className={styles.backButtonLabel}>Back</p>
-  </button>
-);
+const BackButton: FunctionComponent<BackButtonProps> = ({ onClick }) => {
+  const [onHover, setOnHover] = useState(false);
+
+  return (
+    <button
+      onClick={onClick}
+      onMouseEnter={() => setOnHover(true)}
+      onMouseLeave={() => setOnHover(false)}
+      className={styles.backButton}
+    >
+      <ArrowLeftIcon
+        color={onHover ? "#402d28bb" : theme.palette.secondary.main}
```

Comment:
Using javascript for hover effects is a bad practice. I would suggest something like:
```css
.backButton {
  &:hover .arrowIcon {
    color: #402d28bb;
  }
  
  &:hover .backButtonLabel {
    /* Your hover styles for label */
  }
}
```